From 5161dfd03769bee41e62a5d2a3dd64ad868f8819 Mon Sep 17 00:00:00 2001
From: Felipe Contreras <felipe.contreras@gmail.com>
Date: Tue, 23 Mar 2010 21:25:36 +0000
Subject: [PATCH 026/135] dsp-bridge: deh: improve logging stuff

Get rid of printk, use dev_foo, and prettify the output.

Signed-off-by: Felipe Contreras <felipe.contreras@gmail.com>
---
 drivers/dsp/bridge/wmd/ue_deh.c |   33 ++++++++++++---------------------
 1 files changed, 12 insertions(+), 21 deletions(-)

diff --git a/drivers/dsp/bridge/wmd/ue_deh.c b/drivers/dsp/bridge/wmd/ue_deh.c
index 47d3aa3..677012c 100644
--- a/drivers/dsp/bridge/wmd/ue_deh.c
+++ b/drivers/dsp/bridge/wmd/ue_deh.c
@@ -189,9 +189,7 @@ void bridge_deh_notify(struct deh_mgr *hdeh_mgr, u32 ulEventMask, u32 dwErrInfo)
 					&resources);
 
 	if (MEM_IS_VALID_HANDLE(deh_mgr_obj, SIGNATURE)) {
-		printk(KERN_INFO
-		       "bridge_deh_notify: ********** DEVICE EXCEPTION "
-		       "**********\n");
+		dev_info(bridge, "%s: device exception\n", __func__);
 		dev_context =
 		    (struct wmd_dev_context *)deh_mgr_obj->hwmd_context;
 
@@ -203,26 +201,20 @@ void bridge_deh_notify(struct deh_mgr *hdeh_mgr, u32 ulEventMask, u32 dwErrInfo)
 			deh_mgr_obj->err_info.dw_val2 = 0L;
 			deh_mgr_obj->err_info.dw_val3 = 0L;
 			deh_mgr_obj->err_info.dw_val1 = dwErrInfo;
-			printk(KERN_ERR
-			       "bridge_deh_notify: DSP_SYSERROR, err_info "
-			       "= 0x%x\n", dwErrInfo);
+			dev_err(bridge, "%s: %s, err_info = 0x%x\n",
+					__func__, "DSP_SYSERROR", dwErrInfo);
 			break;
 		case DSP_MMUFAULT:
 			/* MMU fault routine should have set err info
 			 * structure */
 			deh_mgr_obj->err_info.dw_err_mask = DSP_MMUFAULT;
-			printk(KERN_INFO "bridge_deh_notify: DSP_MMUFAULT,"
-			       "err_info = 0x%x\n", dwErrInfo);
-			printk(KERN_INFO
-			       "bridge_deh_notify: DSP_MMUFAULT, High "
-			       "Address = 0x%x\n",
-			       (unsigned int)deh_mgr_obj->err_info.dw_val1);
-			printk(KERN_INFO "bridge_deh_notify: DSP_MMUFAULT, Low "
-			       "Address = 0x%x\n",
-			       (unsigned int)deh_mgr_obj->err_info.dw_val2);
-			printk(KERN_INFO
-			       "bridge_deh_notify: DSP_MMUFAULT, fault "
-			       "address = 0x%x\n", (unsigned int)fault_addr);
+			dev_info(bridge, "%s: %s, err_info = 0x%x\n",
+					__func__, "DSP_MMUFAULT", dwErrInfo);
+			dev_info(bridge, "%s: %s, high=0x%x, low=0x%x, "
+				"fault=0x%x\n", __func__, "DSP_MMUFAULT",
+				(unsigned int) deh_mgr->err_info.dw_val1,
+				(unsigned int) deh_mgr->err_info.dw_val2,
+				(unsigned int) fault_addr);
 			dummy_va_addr =
 			    (u32) mem_calloc(sizeof(char) * 0x1000, MEM_PAGED);
 			mem_physical =
@@ -261,9 +253,8 @@ void bridge_deh_notify(struct deh_mgr *hdeh_mgr, u32 ulEventMask, u32 dwErrInfo)
 			deh_mgr_obj->err_info.dw_val2 = 0L;
 			deh_mgr_obj->err_info.dw_val3 = 0L;
 			deh_mgr_obj->err_info.dw_val1 = dwErrInfo;
-			printk(KERN_ERR
-			       "bridge_deh_notify: DSP_PWRERROR, err_info "
-			       "= 0x%x\n", dwErrInfo);
+			dev_err(bridge, "%s: %s, err_info = 0x%x\n",
+					__func__, "DSP_PWRERROR", dwErrInfo);
 			break;
 #endif /* CONFIG_BRIDGE_NTFY_PWRERR */
 		default:
-- 
1.7.0.4

